Assets/MRTK/Providers/LeapMotion/Editor/LeapMotionConfigurationChecker.cs

```diff
@@ -158,6 +169,32 @@ private static bool LeapCoreAssetsVersionSupport()
                         if (line.Contains(versionNumberSupported))
                         {
                             currentLeapCoreAssetsVersion = versionNumberSupported;
+
+                            // The Leap Motion Unity modules Version.txt has remained 4.5.1 across versions 4.6.0, 4.7.0 and 4.7.1, check for the presence
+                            // of certian paths to infer the version number.
+
+                            // This path is only present in 4.7.1
+                            string leap471Path = Path.Combine(Application.dataPath, pathDifference, leapAsmDefPath_471);
+
+                            // This path is present in versions 4.7.0 and 4.7.1
+                            string testDirectoryPath = Path.Combine(Application.dataPath, pathDifference, leapTestsPath_470);
+
+                            // This path is present in 4.6.0 and not 4.5.1
+                            string xrPath = Path.Combine(Application.dataPath, pathDifference, leapXRPath_451);
```

Comment:
a bit of a nit, but if 

> This path is present in 4.6.0 and not 4.5.1

should it be named `leapXRPath_460`?

Reply:
That names does make more sense. Updated.